charts/stable/common/values.yaml

```diff
@@ -253,6 +260,16 @@ persistence:
       enabled: true
     mountPath: /shared
 
+# Mount a folder on the host directly to the containers
+# hostPathMounts:
+#   - name: "data"
+#     enabled: false
+#     emptyDir: false
```

Comment:
emptyDir is not really applicable for hostPathMounts right?

Reply:
It's an option that is actually parsed and might be usefull in some testing scenario's.